AndroidIDPTemplate/app/build.gradle

```diff
@@ -2,13 +2,13 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 
 dependencies {
-  implementation project(':libs:MobileSync')
+    implementation 'com.salesforce.mobilesdk:MobileSync'
```

Comment:
@wmathurin - I opted to remove the version specifier so perhaps it is clearer this will be resolved from the local contents of the Gradle composite build via `includeBuild`.